website/source/docs/platform/k8s/run.html.md

```diff
@@ -0,0 +1,234 @@
+upgrade, etc. the Consul cluster.
+
+The Helm Chart has no required configuration and will install a Consul
+cluster with sane defaults out of the box. Prior to going to production,
+it is highly recommended that you
+[learn about the configuration options](/docs/platform/k8s/helm.html#configuration-values-).
+
+## How-To
+
+### Installing Consul
+
+To install Consul, clone the consul-helm repository, checkout the latest release, and install
+Consul. You can run `helm install` with the `--dry-run` flag to see the
+resources it would configure. In a production environment, you should always
+use the `--dry-run` flag prior to making any changes to the Consul cluster
+via Helm.
+
+```sh
+# Clone the chart repo
+$ git clone https://github.com/hashicorp/consul-helm.git
+$ cd consul-helm
+
+# Checkout a tagged version
+$ git checkout v0.1.0
+
+# Run Helm
+$ helm install --name consul ./
+...
+```
+
+_That's it._ The Helm chart does everything to setup a recommended
+Consul-on-Kubernetes deployment.
+In a couple minutes, a Consul cluster will be formed and a leader
+elected and every node will have a running Consul agent.
+
+The defaults will install both server and client agents. To install
+only one or the other, see the
+[chart configuration values](/docs/platform/k8s/helm.html#configuration-values-).
+
+### Viewing the Consul UI
+
+The Consul UI is enabled by default when using the Helm chart.
+For security reasons, it isn't exposed via a Service by default so you must
+use `kubectl port-forward` to visit the UI. Once the port is forwarded as
+shown below, navigate your browser to `http://localhost:8500`.
+
+```
+$ kubectl port-forward consul-server-0 8500:8500
+...
+```
+
+The UI can also be exposed via a Kubernetes Service. To do this, configure
+the [`ui.service` chart values](/docs/platform/k8s/helm.html#v-ui-service).
+
+### Joining an Existing Consul Cluster
+
+If you have a Consul cluster already running, you can configure your
+Kubernetes nodes to join this existing cluster.
+
+```yaml
+global:
+  enabled: false
+
+client:
+  enabled: true
+  join:
+    - "provider=my-cloud config=val ..."
+```
+
+The `values.yaml` file to configure the Helm chart sets the proper
+configuration to join an existing cluster.
+
+The `global.enabled` value first disables all chart components by default
+so that each component is opt-in. This allows us to _only_ setup the client
+agents. We then opt-in to the client agents by setting `client.enabled` to
+`true`.
+
+Next, `client.join` is set to an array of valid
+[`-retry-join` values](/docs/agent/options.html#retry-join). In the
+example above, a fake [cloud auto-join](/docs/agent/cloud-auto-join.html)
+value is specified. This should be set to resolve to the proper addresses of
+your existing Consul cluster.
+
+-> **Networking:** Note that for the Kubernetes nodes to join an existing
+cluster, the nodes (and specifically the agent pods) must be fully connected.
```

Comment:
Not clear if you mean the Agent pods need to be connected _just_ to each other which is default in k8s. Suggest maybe:

> the nodes (and specifically the agent pods) must be able to connect to all server and client agents inside _and outside_ of kubernetes.